### DISCUSSION TITLE
How to update directorate review after the authors have made the update

### DISCUSSION THREAD

**Discussion by dhruvdhody** [ietf-tools/datatracker#8215] [Help / Questions] [answered]
### Description

How to update directorate review after the authors have made the update? Should I be using "correct review" button such as - https://datatracker.ietf.org/doc/draft-ietf-bier-ping/reviewrequest/14442/complete/ to go and update my older review comments to now say it is okay! How do I maintain the history of older comments that I provided? 

### Code of Conduct

- [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment, marked as answer]
This is currently a small mess.

The best thing to do is to revise the review, keeping the original content intact, but prefixing or inlining with statements that the review is complete. There won't be a diff back to the original review.

See #6941 (which will serve as the bug report with this as a mention).

As this is more of a "how do I use this" question, I'm moving it to a discussion.

**Reply by Nika78s** [reply]
#- [ ] -

**Comment by d121122gma** [comment]
> ### Description
> 
> How to update directorate review after the authors have made the update? Should I be using "correct review" button such as - https://datatracker.ietf.org/doc/draft-ietf-bier-ping/reviewrequest/14442/complete/ to go and update my older review comments to now say it is okay! How do I maintain the history of older comments that I provided? 
> 
> ### Code of Conduct
> 
> - [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by d121122gma** [comment]
> ### Description
> 
> How to update directorate review after the authors have made the update? Should I be using "correct review" button such as - https://datatracker.ietf.org/doc/draft-ietf-bier-ping/reviewrequest/14442/complete/ to go and update my older review comments to now say it is okay! How do I maintain the history of older comments that I provided? 
> 
> ### Code of Conduct
> 
> - [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
Locking this conversation as it's just attracting off-topic comments and spam.
